src/util/moneystr.cpp

```diff
@@ -37,7 +37,7 @@ bool ParseMoney(const std::string& str, CAmount& nRet)
         return false;
     }
 
-    if (str.empty()) {
```

Comment:
OK, I would prefer to do a `str = TrimString(str, " ")` here, to at least avoid going over the spaces twice!
(and keep the empty-check as it, after that)

Reply:
The signature needs to be changed to `bool ParseMoney(std::string str, CAmount& nRet)`